### ISSUE TITLE
Missing Data to Duplicate the Experiments

### ISSUE THREAD

**Issue by redraven984** [VLL-HD/FrEIA#23]
It looks like I can't duplicate the experiments here because I'm missing the data. I'm specifically interested in inverse_problems_science. Could you please point me to this data - or tell me how to format the data in the expected format?

Thanks!

**Comment by Johnson-yue** [comment]
Q1: did you download the LSUN_bedroom dataset？？
A2: the format of data is convert source color image from RGB to YCrCb， and using CrCb  to represent  color ， the condition is Y and output is CrCb channel

**Comment by ardizzone** [comment]
You are right, there is no dataset from our collaborators that did the medical example (the person responsible has since left the group)
I will try to ask around and get this sorted!

I am moving this issue to the paper's own repository (this is now only for the FrEIA itself, which can implement arbitrary reversible architectures and training schemes, not just our method).
